eventuate-local-java-embedded-cdc/src/test/java/io/eventuate/local/cdc/debezium/AbstractTopicRelayTest.java

```diff
@@ -66,8 +73,41 @@ public void shouldCaptureAndPublishChange() throws ExecutionException, Interrupt
     long endTime = System.currentTimeMillis();
 
     logger.debug("got the event I just published in msecs {}", endTime - publishTime);
+  }
+
+  @Test
+  public void testDelete() throws ExecutionException, InterruptedException {
 
-//    eventTableChangesToAggregateTopicRelay.stopCapturingChanges();
+    String aggregateType = "TestAggregate" + UUID.randomUUID().toString();
+    String eventType = "TestEvent" + UUID.randomUUID().toString();
+
+    List<EventTypeAndData> myEvents = Collections.singletonList(new EventTypeAndData(eventType, "{}", Optional.empty()));
+
+    EntityIdVersionAndEventIds ewidv = AsyncUtil.await(eventuateJdbcEventStore.save(aggregateType, myEvents, Optional.empty()));
+
+    Set<Int128> expectedEventIds = new HashSet<>();
```

Comment:
Copy/pasting this code is not good.
This should be packaged into a TestSubscriber and TestSubscriberFactory so that a test can go:

```
TestSubscriber subscriber = testSubscriberFactory.subscribe(subscriber, typesAndEvents, expectedIds)

subscriber.assertReceived()
subscriber.assertNotReceived() // assuming that this is useful.
```

Reply:
ok